.github/workflows/build-test.yaml

```
@@ -21,18 +22,22 @@ jobs:
       - name: Set up go
         timeout-minutes: 5
-        uses: actions/setup-go@v6
+        uses: actions/setup-go@4a3601121dd01d1626a1e23e37211e3254c1c06c # v6
         with:
           go-version-file: "go.mod"
 
       - name: Lint
-        uses: golangci/golangci-lint-action@v9
+        uses: golangci/golangci-lint-action@1e7e51e771db61008b38414a730f564565cf7c20 # v9
         with:
+<<<<<<< HEAD
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)